### ISSUE TITLE
Update suggestion?

### ISSUE THREAD

**Issue by verveguy** [toffer/minecloud-ami#17]
I've pulled a few of the changes from dmotles into my fork. Makes me realize that, short of building a whole new AMI, I'm not clear on recommended practice for updating the deployed instance. Did I miss something in the notes / obvious ?

**Comment by toffer** [comment]
No, rebuilding the AMI is how I was thinking of handling updates. The instances aren't long-running, so I figure it's better to rebuild, rather than update. That way, you don't have to run something like `puppet apply` when you boot up an instance.
